inttest/k0sctl/k0sctl_test.go

```diff
@@ -113,8 +113,9 @@ func (s *K0sctlSuite) k0sctlInitConfig() map[string]interface{} {
 
 	for _, m := range machines {
 		port, err := m.HostPort(22)
-		s.NoError(err)
-		addresses = append(addresses, fmt.Sprintf("127.0.0.1:%d", port))
+		if s.NoError(err) {
```

Comment:
why not to change to require and fail fast?

Reply:
I probably just saw that it's a loop and thought that it'd be good to check all of the items in there. But yeah, seems to be a bit pointless to run the test with only part of the machines. Will change.